src/auth_manager/auth_manager.cpp

```diff
@@ -40,16 +40,15 @@ AuthVerifyStatus AuthManager::verify(const std::string& token, std::string& msg)
     // if we have it in cache, just use it to make the decision
     auto const token_hash = md5_sum(token);
     if (auto const ct = m_cached_tokens.get(token_hash); ct) {
-        auto const& cached_token = ct->get();
-        if (cached_token.valid) {
+        if (ct->valid) {
```

Comment:
Shouldn't this just exit the scope and get a new token as if it wasn't there? How does an expired token get refreshed rather than just rotated around the cache?

Reply:
This code path is in server side which verifies the token, if the token is expired, we just sent an error code back, something like unauthorized.